doc/examples/edges/plot_convex_hull.py

```diff
@@ -18,38 +15,23 @@
 import matplotlib.pyplot as plt
 
 from skimage.morphology import convex_hull_image
+from skimage import data
+from skimage.util import invert
 
-
-image = np.array(
-    [[0, 0, 0, 0, 0, 0, 0, 0, 0],
-     [0, 0, 0, 0, 1, 0, 0, 0, 0],
-     [0, 0, 0, 1, 0, 1, 0, 0, 0],
-     [0, 0, 1, 0, 0, 0, 1, 0, 0],
-     [0, 1, 0, 0, 0, 0, 0, 1, 0],
-     [0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=float)
-
-original_image = np.copy(image)
+# The original image is inverted as the object must be white.
```

Comment:
But Convex-hull works only on binary images. Do you think that "higher" is appropriated?